src/Shortcodes/FileLinkTracking.php

```diff
@@ -131,15 +131,21 @@ public function augmentSyncLinkTracking()
         $allFields = DataObject::getSchema()->fieldSpecs($this->owner);
         $linkedPages = [];
         $anyBroken = false;
+        $hasTrackedFields = false;
         foreach ($allFields as $field => $fieldSpec) {
             $fieldObj = $this->owner->dbObject($field);
             if ($fieldObj instanceof DBHTMLText) {
+                $hasTrackedFields = true;
                 // Merge links in this field with global list.
                 $linksInField = $this->trackLinksInField($field, $anyBroken);
                 $linkedPages = array_merge($linkedPages, $linksInField);
             }
         }
 
+        if (!$hasTrackedFields) {
```

Comment:
Please add a comment here about why we don't just check `$linkedPages` - I was about to suggest the same thing that @michalkleiner had suggested in https://github.com/silverstripe/silverstripe-assets/pull/558#r1238430544

Reply:
done